apps/src/feedback.js

```diff
@@ -1055,10 +1055,10 @@ FeedbackUtils.prototype.createModalDialogWithIcon = function(options) {
 /**
  * Check for '???' instead of a value in block fields.
  */
-FeedbackUtils.prototype.hasQuestionMarksInNumberField_ = function () {
+FeedbackUtils.prototype.hasQuestionMarksInNumberField = function () {
   return Blockly.mainBlockSpace.getAllBlocks().some(function(block) {
     return block.getTitles().some(function(title) {
-      return title.value_ === '???';
+      return title.value_ === '???' || title.text_ === '???';
```

Comment:
We could. Such a constant would need to be shared across blockly/blockly-core. There may be some value in doing this, but I'm not sure it's that important and I have other things I'd rather do right now.